R/params.R

```diff
@@ -175,6 +175,13 @@ params_configurable <- function(param) {
   length(param$value) <= 1 && !is.null(params_get_control(param))
 }
 
+# Returns a new empty named list.
+params_namedList <- function() {
+  empty <- list()
+  names(empty) <- character()
+  empty
+}
```

Comment:
No. What you did here is basically what setNames() does (take a look at its source code -- it is only two lines). As I said, it is only FYI. It is no big deal to reinvent a wheel of two lines :)